index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    <!--NAV VAR-->
+    <header>
+        <nav>
+            <p>My logo</p>
+            <button class="menu"><img src="Img/Icon - Menu.png" alt=""></button>
+                <section class="menudesk">
+                    <a href="">Portfolio</a>
+                    <a href="">About</a>
+                    <a href="">Contact</a>
+                </section>
+        </nav>
+    </header>
+    <br>
+    <section class="info">
+        <div class="desc">
+            <h1>I'm Andrea.<br>Glad to see you!</h1>
+            <h2>I'm Andrea.<br>Glad to see you!</h2>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <div class="enlaces">
+                <p>Let's conect</p>
+                <a href=""><img src="Img/twitter.svg" alt=""></a>
+                <a href=""><img src="Img/Linkedin icon.svg" alt=""></a>
+                <a href=""><img src="Img/medium.svg" alt=""></a>
+                <a href=""><img src="Img/github.svg" alt=""></a>
+                <a href=""><img src="Img/angellist.svg" alt=""></a>
```

Comment:
- [ ] Kindly use an unordered list to list out your social icons. This is best for HTML semantics. The list of social icons should look like this;
```html
<ul>
  <li>
    <a><img /> </a>
  </li>
  <li>
    <a><img /> </a>
  </li>
  <li>
    <a><img /> </a>
  </li>
</ul>
```

- [ ] Kindly add some space between the social icons to match the Figma template. You are expected to stick to design as much as possible. This includes fonts, sizes, colors, and spacings.
### Figma 
![image](https://user-images.githubusercontent.com/94551484/234983999-1169ef6c-4760-4e0e-a445-60181a664b88.png)
### Your design\
![image](https://user-images.githubusercontent.com/94551484/234984122-521efd80-73c3-4ae4-a89e-44af9b6ae3f0.png)